### DISCUSSION TITLE
taro react-hook h5 没有找到页面实例。

### DISCUSSION THREAD

**Discussion by kaifa-yang** [NervJS/taro#12495] [Q & A] [answered]
### 相关平台
微信小程序

### 复现仓库
[https://github.com/kaifa-yang/taro-react-.git](https://github.com/kaifa-yang/taro-react-.git)
**小程序基础库: 1.0.0**
**使用框架: React**

### 复现步骤
终端 npm run dev:h5
运行或者刷新就会碰到错误

### 期望结果
正常运行，不会再出现没有找到页面实例

### 实际结果
目前无法解决运行刷新就报错-且同时无法渲染
得在代码中任意改动保存-->不刷新->能刷新

### 环境信息
```
Taro CLI 3.5.5 environment info:
    System:
      OS: Windows 10
    Binaries:
      Node: 16.13.2 - D:\node.EXE
      Yarn: 1.22.17 - C:\Users\人生赢家\AppData\Roaming\npm\yarn.CMD
      npm: 8.1.2 - D:\npm.CMD
```

**Comment by kaifa-yang** [comment]
救救孩子，已经碰壁几天了。

**Comment by TheKonka** [comment]
#12416  也是报 这个错误

**Comment by ZakaryCode** [comment, marked as answer]
入口文件书写错误，不需要传入页面组件

```
function App(props) {
  // props.children 是将渲染的页面实例，缺少将导致错误
  return props.children
}

export default App
```
